components/psst/common/pref_names.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_PSST_COMMON_PREF_NAMES_H_
+#define BRAVE_COMPONENTS_PSST_COMMON_PREF_NAMES_H_
+
+#include "base/component_export.h"
+#include "components/prefs/pref_service.h"
```

Comment:
this is the wrong import, you either need pref_registry_simple.h or a forward decl